### DISCUSSION TITLE
Allowing the possibility to add external extensions of quests

### DISCUSSION THREAD

**Discussion by severinmenard** [streetcomplete/StreetComplete#5626] [Ideas]
JOSM has default presets but also allows its users to add new ones, either from a open list in the JOSM server, or from a local file. I would suggest to implement the equivalent in StreetComplete i.e the capacity to add external extensions of quests. 
This would:
- avoid forks like https://github.com/Helium314/SCEE
- allow StreetComplete to be used for specific uses and contexts, e.g. evaluating building damages after floods or earthquake

**Comment by Helium314** [comment]
I don't think this will happen, see e.g. #98 and #3959

> avoid forks like https://github.com/Helium314/SCEE

No it would not. Check the list of differences to SC and you will see there is much more than just quests.

**Reply by severinmenard** [reply]
Thank you for the links, I commented the most recent one.

Yes now there is more than quests. Once you have forked, there projects will be more and more different as time goes. Ex Maps.me/Organic Maps

**Reply by Helium314** [reply]
I think an important difference between SC/SCEE and Maps.me/OrganicMaps is that SCEE is still always based on the latest version of SC, so all new SC features will go into SCEE.
